source/common/event/dispatcher_impl.cc

```diff
@@ -37,31 +37,180 @@
 namespace Envoy {
 namespace Event {
 
+DispatcherImplBase::DispatcherImplBase(Api::Api& api, TimeSystem& time_system,
```

Comment:
To make this review simpler can you just rename things in place? If you want to move them later that's fine but for now it's hard to tell what is new/old code.